fcl/dunefdvd/gen/solar/prodmarley_flat_dunevd10kt_1x8x6_3view_30deg.fcl

```diff
@@ -0,0 +1,5 @@
+#include "prodmarley_nue_spectrum_dunevd10kt_1x8x6_3view_30deg.fcl"
+
+physics.producers.marley: @local::dune_marley_flat
+physics.producers.marley.marley_parameters.source.Emax: 30.
+physics.producers.marley.marley_parameters.source.E_bin_lefts: [2.]
```

Comment:
These should inherit from an upstream fcl

Reply:
Hi! What do you think would be the best fcl? marley_dune.fcl?

Reply:
For now it was set in one of the running fhicls and all other inherit from it. But, yes, the ideal would be to set this is marley_dune.fcl.